sdk/digitaltwins/Azure.DigitalTwins.Core/src/Generated/DigitalTwinModelsRestClient.cs

```diff
@@ -88,14 +88,7 @@ public async Task<Response<PagedModelDataCollection>> ListAsync(IEnumerable<stri
                     {
                         PagedModelDataCollection value = default;
                         using var document = await JsonDocument.ParseAsync(message.Response.ContentStream, default, cancellationToken).ConfigureAwait(false);
-                        if (document.RootElement.ValueKind == JsonValueKind.Null)
```

Comment:
We have this same pattern copied into our customized code. Will this PR update those as well?

example: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/ModelData.Serialization.cs#L28

Reply:
No, the generator won't update the manually copied code. Having an extra null check doesn't hurt but as part of tightening the nullability rules we tried to keep them to a minimum in the generated code.

Reply:
So we should update our customized code to match when we have the opportunity?

Reply:
Yes. I'll add this to the description that I use for these PRs.